### PR TITLE
Aml 3946 email preview

### PR DESCRIPTION
**Pull request by Howesy77**

### CODE REVIEW

File: src/SFA.DAS.ProviderRegistrations.UnitTests/Application/Queries/GetProviderByUkprnQueryHandlerTests.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        {
+            //arrange
+            var query = new GetProviderByUkprnQuery(ukprn);
+
+            //act
+            var result = await handler.Handle(query, new CancellationToken());
+
+            //assert
+            result.Should().NotBeNull();
+            result.ProviderName.Should().NotBeNullOrWhiteSpace();
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.ProviderRegistrations.UnitTests/Application/Queries/GetProviderByUkprnQueryHandlerTests.cs]: I'm not overly familiar with AutoData...but can we make the assertion more specific so it verifies the expected provider name is returned?

File: src/SFA.DAS.ProviderRegistrations/Application/Queries/GetProviderByUkprn/GetProviderByUkprnQueryResult.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+using SFA.DAS.Apprenticeships.Api.Types.Providers;
+
+namespace SFA.DAS.ProviderRegistrations.Application.Queries.GetProviderByUkprn
+{
+    public class GetProviderByUkprnQueryResult
+    {
+        public GetProviderByUkprnQueryResult(Provider provider)
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.ProviderRegistrations/Application/Queries/GetProviderByUkprn/GetProviderByUkprnQueryResult.cs]: I don't think I like the Result object having a dependency on the API type

Comment (Howesy77) [reply src/SFA.DAS.ProviderRegistrations/Application/Queries/GetProviderByUkprn/GetProviderByUkprnQueryResult.cs]: Changed

File: src/SFA.DAS.ProviderRegistrations.UnitTests/Application/Queries/GetProviderByUkprnQueryHandlerTests.cs (hunk 1 above)
Comment (Howesy77) [reply src/SFA.DAS.ProviderRegistrations.UnitTests/Application/Queries/GetProviderByUkprnQueryHandlerTests.cs]: No idea. Can we wait for Chris to return?

Comment (MichaelYoung1981) [reply src/SFA.DAS.ProviderRegistrations.UnitTests/Application/Queries/GetProviderByUkprnQueryHandlerTests.cs]: Sure

**Pull request merged**